frontend/src/components/DataExportButton.vue

```diff
@@ -181,6 +181,7 @@ const tryExportViaForm = async (e: MouseEvent) => {
   formRef.value?.validate((errors) => {
     if (errors) return;
     doExport(formData.value.format, formData.value.limit);
+    state.showDrawer = false;
```

Comment:
Looks like we should add this ad line 216 under the `emit("export")`

Reply:
Was confused about two difference options with tryExportViaDropdown() and tryExportViaForm(). Addressed it by moving to the finally block.

Reply:
I was also thinking whether close the drawer before or after user clicks the confirm download button. Neither is perfect. Let's close it after the download for now.